docs/index.rst

```diff
@@ -0,0 +1,40 @@
+Welcome to betterproto's documentation!
+=======================================
+
+betterproto is a protobuf compiler and interpreter. It was written for the modern age making use of
+Python 3.6+ features to generate readable, understandable and idiomatic Python code.
+
+
+Features:
+~~~~~~~~~
```

Comment:
The logical flow of the features section is a bit unclear to me. What I think is important is that someone who just landed here from google who doesn't know what betterproto is about, gets a clear idea of the scope of the library and why it's a good choice.

How about:

```
Idiomatic code generation for Protobuf 3 & gRPC
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- Python source code is generated as python packages with the same structure as the source protobuf packages
- Message types are modelled as python dataclasses with support for binary as JSON serialisation
- Message contents are represented using the relevant python type such as enum, datetime, or timedelta
- Generated code includes complete type annotations
- Generated gRPC clients support async/await
```

Reply:
Does this seem better?
```
- Generated messages are both binary & JSON serializable
- Messages use relevant python types, e.g. ``Enum``, ``datetime`` and ``timedelta`` objects
- ``async``/``await`` support for gRPC Clients
- Relative imports, mocking protobuf package structure
- Mypy type checkable with full type annotations
- Modern idiomatic python naming conventions
```

Reply:
It's better. As a feature list without much introduction I feel it needs to say things in a way that doesn't assume too much context.

I think it's worth mentioning dataclasses.
I'm not totally sure what you mean about "naming conventions".
I don't feel "Relative imports" per se is that interesting a feature so much as that the output is a valid python package with a logical structure (unlike the standard compiler). I wouldn't use the word mocking in this context.